Brain/Brain.ino

```diff
@@ -58,6 +58,7 @@ const char * file_name = str_file_name.c_str();
 
 long startTime;
 String packet;
+String error_packet;
```

Comment:
don't think we need a separate variable for this; just use packet. It's a transient variable anyway, it changes on every iteration